libraries/AP_HAL_ChibiOS/hwdef/PixPilot-V6PRO/README.md

```diff
@@ -0,0 +1,352 @@
+=========================
+
+TELEM1, TELEM2 ports
+--------------------
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>TX (OUT)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>RX (IN)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+I2C1, I2C2 ports
+---------------
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>PIN</th>
+   <th>SIGNAL</th>
+   <th>VOLT</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>SCL</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>SDA</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+CAN1, CAN2 ports
+---------------
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>PIN</th>
+   <th>SIGNAL</th>
+   <th>VOLT</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>CAN_H</td>
+   <td>+12V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>CAN_L</td>
+   <td>+12V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+Safety and buzzer port
+-----------
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>PIN</th>
+   <th>SIGNAL</th>
+   <th>VOLT</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>LED</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>SAFKEY</td>
+   <td>+5V</td>
+   </tr>
+   </tbody>
+   </table>
+
+GPS1/I2C1, GPS2/I2C2 ports
+--------------------------
+
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>PIN</th>
+   <th>SIGNAL</th>
+   <th>VOLT</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>TX</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>RX</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>SCL</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>5</td>
+   <td>SDA</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+Serial5 port
+--------------------
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>TX (OUT)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>RX (IN)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+Power1, Power2 ports
```

Comment:
Missing the CAN power ports, and DSM port

Reply:
Also where are the two analog inputs PC3/4?

Reply:
I have added  CAN power ports and DSM port。 the two analog inputs PC3/4 on the power2 ports.